.github/workflows/snapshot.yaml

```diff
@@ -22,3 +22,24 @@ jobs:
 
       - name: Build and test
         run: ./gradlew -Psnapshot=yes build --scan --no-daemon
```

Comment:
> `-Psnapshot=yes`

Should we change how that flag works? Perhaps we should default to `snapshot=yes` and only pass `snapshot=no` when releasing a new stable version?

Reply:
I am not sure... Let's leave that as a separate though